### PR TITLE
Added elements for creating text boxes

### PR DESCRIPTION
**Pull request by cacqw7**

### CODE REVIEW

**Comment by boblail** [comment]
Why do you **Add VML elements for text boxes**? 

If we don't generate the unprintable mailing label outlines with VML, is there any reason for us to implement/support VML?

**Comment by cacqw7** [comment]
This PR started as VML, but I didn't realize that VML wouldn't work for rendering text boxes until I'd already implemented all the elements. I can move the VML pieces into a different PR.

**Comment by cacqw7** [comment]
I pulled out the VML elements into #48. This PR is ready for re-review.

File: lib/openxml/docx/elements/word_processing_drawing_anchor.rb [hunk 1]
```diff
@@ -5,17 +5,17 @@ class WordProcessingDrawingAnchor < Container
...
         attribute :distance_from_top, expects: :positive_integer, displays_as: :distT         # Optional
-        attribute :hidden, expects: :true_or_false                                            # Optional
-        attribute :layout_in_cell, expects: :true_or_false, displays_as: :layoutInCell        # Required
-        attribute :locked, expects: :true_or_false                                            # Required
+        attribute :hidden, expects: :boolean                                            # Optional
+        attribute :layout_in_cell, expects: :boolean, displays_as: :layoutInCell        # Required
+        attribute :locked, expects: :boolean                                            # Required
         attribute :z_index, expects: :positive_integer, displays_as: :relativeHeight          # Required
-        attribute :simple_position, expects: :true_or_false, displays_as: :simplePos          # Optional
+        attribute :simple_position, expects: :boolean, displays_as: :simplePos          # Optional
```
Comment (boblail) [inline lib/openxml/docx/elements/word_processing_drawing_anchor.rb]: After this find-and-replace, do you want to fix the indenting?

File: lib/openxml/docx/properties/columns.rb [hunk 2]
```diff
@@ -8,11 +8,18 @@ class Columns < ContainerProperty
...
           attribute :number, expects: :integer, displays_as: :num
-          attribute :separator, expects: :true_or_false, displays_as: :sep
+          attribute :separator, expects: :boolean, displays_as: :sep
           attribute :space, expects: :integer
         end
+
+        def render?
+          return true if %i(equal_width number separator space).any? { |attribute| !public_send(attribute).nil? }
+          super
+
```
Comment (boblail) [inline lib/openxml/docx/properties/columns.rb]: No need for this blank line

**Pull request merged**